src/components/Button.tsx

```diff
@@ -0,0 +1,11 @@
+import Button from '@mui/material/Button';
+
+const BasicButtons = ({text}:any) => {
+  return (
+    <Button variant="contained">
```

Comment:
add color for changing button betweem primary and secondary

Reply:
done